delivery/check-issue-exists/action.yml

```diff
@@ -28,12 +34,18 @@ runs:
             }
             `,
             { issueId: githubIssueId}
-          )
+          ).catch(error => {
+            core.info(`Error "${error.message}" while trying to find the issue with ID: ${githubIssueId}`);          
```

Comment:
you can use `core.error` in that case

Reply:
core.error trigger a workflow failure, while here I only want this as an info

Reply:
Updated message to remove the mention of "error", since trying to fetch a missing issue is not necessarily an error.